src/docs/reference/backups.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Backups
+---
+
+Backups are a feature that enables point in time data recovery for [volumes](/reference/volumes) on Railway.
+
+## How it works
+
+When mounted to a service, backups can be manually created, deleted and restored. And they can also work on a Daily/Weekly/Monthly schedule.
+
+Schedules are beta and exist behind a feature-flag attached to the service.
+
+## Retention
```

Comment:
Regardless of whether we have plans to do it, I think you should answer the question "What if I want hourly backups".

Reply:
How can I phrase that in a way that states that hourly backups are not available unless... you are GoCo or other high-commited spender and really wants it.

Reply:
Say that it would be an enterprise feature and link to demo new?

Reply:
What is demo new?

And I don't want enterprises they have it ready, it's something they should negotiate and pay more for. Or be so important that we care.

Reply:
We link to it in a hand full of places - https://cal.com/team/railway/demonew

But yeah good point, then I guess we leave it up to the user to reach out to us about it.

And even so, the template allows you to do minutely backups, so it's not like it's not possible.